libbpfgo.go

```diff
@@ -1464,6 +1490,28 @@ func (p *BPFProg) AttachNetns(networkNamespacePath string) (*BPFLink, error) {
 	return bpfLink, nil
 }
 
+type IterOpts struct {
+	MapFd int
+}
+
+func (p *BPFProg) AttachIter(opts IterOpts) (*BPFLink, error) {
```

Comment:
use opts to support the upcoming libbpf v1.0.2:
https://github.com/aquasecurity/libbpfgo/pull/254/commits/bd0645b71262eae55682788fa47ee79d6c36bc67
https://github.com/libbpf/libbpf/commit/981001bf468ec5bf4bd9bcbdd378f5748b515c0c